doc/source/user_guide/basics.rst

```diff
@@ -2050,33 +2050,42 @@ The following table lists all of pandas extension types. For methods requiring `
-|                                                 |                           |                    |                               | ``'Sparse[float]'``                    |
-+-------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
-| :ref:`intervals <advanced.intervalindex>`       | :class:`IntervalDtype`    | :class:`Interval`  | :class:`arrays.IntervalArray` | ``'interval'``, ``'Interval'``,        |
-|                                                 |                           |                    |                               | ``'Interval[<numpy_dtype>]'``,         |
-|                                                 |                           |                    |                               | ``'Interval[datetime64[ns, <tz>]]'``,  |
-|                                                 |                           |                    |                               | ``'Interval[timedelta64[<freq>]]'``    |
-+-------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
-| :ref:`nullable integer <integer_na>`            | :class:`Int64Dtype`, ...  | (none)             | :class:`arrays.IntegerArray`  | ``'Int8'``, ``'Int16'``, ``'Int32'``,  |
-|                                                 |                           |                    |                               | ``'Int64'``, ``'UInt8'``, ``'UInt16'``,|
-|                                                 |                           |                    |                               | ``'UInt32'``, ``'UInt64'``             |
-+-------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
-| :ref:`Strings <text>`                           | :class:`StringDtype`      | :class:`str`       | :class:`arrays.StringArray`   | ``'string'``                           |
-+-------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
-| :ref:`Boolean (with NA) <api.arrays.bool>`      | :class:`BooleanDtype`     | :class:`bool`      | :class:`arrays.BooleanArray`  | ``'boolean'``                          |
-+-------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
++---------------------------------------------------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
```

Comment:
Could you post a screenshot of how this looks? It's somewhat hard to see what changed from the diff

Reply:
so, the issue is fixed? Right ! or can i still push my changes? I am new in open source

Reply:
Yes, if interested in contributing you can find an issue that hasn't been assigned yet or doesn't have an open pull request

Reply:
Ok, thank you. So on an issue only one person can pull a request at one time?

Reply:
Ok thanks mroeschke

Reply:
<img width="1144" alt="dtypes_pandas" src="https://user-images.githubusercontent.com/91160475/215066781-7bab57d7-6088-4f61-9c9e-05203dd09923.png">

This is a screenshot of that part of the table. 
As Marco said, there is no need to add NumPy integer and NumPy float to the table of pandas extension types. I corrected the table and left only "nullable float" under "nullable integer" (without the rest). This implies that we shouldn’t resize the table.